scriptmodules/ports/digger.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="digger"
+rp_module_desc="Digger Remastered"
+rp_module_licence="GPL https://raw.githubusercontent.com/sobomax/digger/master/README.md"
+rp_module_section="exp"
+
+function depends_digger() {
+    getDepends cmake libsdl2-dev
+}
+
+function sources_digger() {
+    gitPullOrClone "$md_build" https://github.com/proyvind/digger.git joystick
+}
+
+function build_digger() {
+    cmake . -DCMAKE_INSTALL_PREFIX="$md_inst"
+    make
+    md_ret_require="$md_build"
```

Comment:
This should point to the digger binary so it can send the appropriate return code when the file is missing (i.e. build fails), otherwise it will try to proceed to the install/configure phases erroneously.

Reply:
I noticed a module with this the other day - I think one contributed one had it, then there was some c&p. There are 4 currently like this I am going to fix.